lib/mix/lib/mix/tasks/release.init.ex

```diff
@@ -117,6 +115,7 @@ defmodule Mix.Tasks.Release.Init do
       exec "$REL_VSN_DIR/$REL_EXEC" \
            --cookie "$RELEASE_COOKIE" \
            --$RELEASE_DISTRIBUTION "$RELEASE_NODE" \
+           --erl "-mode $RELEASE_MODE" \
```

Comment:
We have been adding switches to the `elixir` executable but mostly to avoid double encoding paths, otherwise handling paths with spaces or quotes in them is very complicated. This time, I decided to pass it via the `--erl` flag directly but we could add `--mode`. I am fine with both approaches but I am dropping a comment in case someone has stronger feelings about this. Thoughts?

Reply:
If users are going to use this themselves frequently, I'd go with adding `--mode`, otherwise this looks fine to me :)